app/views/whats_new/index.html.erb

```diff
@@ -0,0 +1,75 @@
+<% content_for :page_title, t('whats_new.title') %>
+<% content_for :title, t('whats_new.title') %>
+<% content_for :title_margin_bottom, 0 %>
+
+<%   def render_govspeak(content)
+       render "govuk_publishing_components/components/govspeak" do
+         raw(Govspeak::Document.new(content, sanitize: true).to_html)
+       end
+     end %>
```

Comment:
Rails convention is to define helper methods in Ruby code, so this would be defined in `app/helpers/application_helper.rb`

I do however think that you probably don't need this. It seems quite overkill having a govspeak rendering layer and we could just put HTML directly in